transaction/std/Cargo.toml

```diff
@@ -11,6 +11,7 @@ test-only = []
 # External dependencies
 cfg-if = "1.0"
 displaydoc = { version = "0.2", default-features = false }
+hex = "0.4.3"
```

Comment:
Is this actually needed?

Reply:
ya its not, it was part of the android-rth changes but those aren't in this pr. removed.